### ISSUE TITLE
Keybinding to get the github link for a given line (or block)

### ISSUE THREAD

**Issue by timsampsoncujo** [lewis6991/gitsigns.nvim#429]
**Is your feature request related to a problem? Please describe.**
Often when helping a colleague to debug something I want to say "on line 7 of cache.tl" (using this codebase as an example) which is semi annoying. I think it would be *a lot* clearer and unambiguous to just post a link of the form `https://github.com/lewis6991/gitsigns.nvim/blob/a451f97117bd1ede582a6b9db61c387c48d880b6/teal/gitsigns/cache.tl#L7` or `https://github.com/lewis6991/gitsigns.nvim/blob/a451f97117bd1ede582a6b9db61c387c48d880b6/teal/gitsigns/cache.tl#L7-L24`. It briefly occurred to me to write some lua for this myself but, then I figured gitsigns probably already has a lot of the machinery for something like this (you have git blame and constructing the URL doesn't seem like it would be super difficult - without having thought about it for more than 2mins).

**Describe the solution you'd like**
Basically a keybinding that would produce that link for me.

**Describe alternatives you've considered**
TBH I haven't considered alternatives. Forgive me if this is somewhat presumptuous and perhaps a bit niche, but I thought this may not be too difficult and I saw some branch related to github so assumed that github integration isn't completely off the table.

**Comment by lewis6991** [comment]
I think [gitlinker.nvim](https://github.com/ruifm/gitlinker.nvim) provides exactly what you need.

**Comment by sanga** [comment]
Oh perfect. Thank you
